bigchaindb/common/schema/transaction.yaml

```diff
@@ -105,13 +105,14 @@ definitions:
     description: |
         Description of the asset being transacted. In the case of a ``TRANSFER``
         transaction, this field contains only the ID of asset. In the case
-        of a ``CREATE`` transaction, this field may contain properties:
+        of a ``CREATE`` transaction, this field contains only a user payload
+        associated with the asset.
     additionalProperties: false
-    required:
-    - id
```

Comment:
Looked around to see if we could change the asset model based on `CREATE` and `TRANSFER` transaction but [it'd look pretty ugly](https://github.com/json-schema/json-schema/issues/158#issuecomment-86124035) and might be slower.

Thoughts @libscott? Right now the asset schema is a bit of a lie ;).